### PR TITLE
Remove floating-nav items for existing but empty objects

### PR DESCRIPTION
**Pull request by melo0187**
As I wrote in #49 the floating-nav can contain items for sections that are not rendered because in fact the properties are just there in the json file but without any content.

Maybe adding the isEmpty check is a good idea because it is more flexible? 
In the end it does remove floating-nav items in both cases (property non existing or property is there but empty, like `"awards": []`) so that the floating-nav will always match the actual document.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,9 +10,10 @@ utils.setConfig({
 function interpolate(object, keyPath) {
     var keys = keyPath.split('.');
 
-    return _(keys).reduce(function(res, key) {
+    return !_.isEmpty(
```
Comment (mudassir0909) [inline index.js]: This will change the function definition, please revert this change. You can instead update `getFloatingNavItems` by replacing

```js
     return _(floating_nav_items).filter(function(item) {
         return interpolate(resume, item.requires);
     });
```

with

```js
     return _(floating_nav_items).filter(function(item) {
         var value = interpolate(resume, item.requires);

         return _.isEmpty(value);
     });
```

Which essentially is your change but at a different place

**Comment by mudassir0909** [comment]
Fixed this as part of d278387764e7d3d1e8b013a2823e776c333e2d42

**Comment by melo0187** [comment]
Thanks! Both for the comment and the fix. Much appreciated =)
